src/scripts/utils.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Public: Expand the value from environment variables with prefix.
+#
+# Takes a prefix as an argument and expands the value of the environment variables
+# with the prefix. The expansion is done by using the eval command.
+#
+# $1 - Prefix used to filter the envinronment variables.
+#
+# Examples
+#
+#   expand_env_vars_with_prefix "ORB_PARAM_"
+#   expand_env_vars_with_prefix "PARAM_"
+#
+# Returns nothing.
+expand_env_vars_with_prefix() {
```

Comment:
Would love some thoughts on this function. I'm including it as a solution for the problem found in the migrations of the Docker and the AWS orbs.

Reply:
Which issue?